tldraw/apps/tldraw-logseq/src/styles.css

```diff
@@ -56,22 +56,21 @@
 
 
 .logseq-tldraw .action-bar {
-  position: relative;
-  top: 0;
+  position: absolute;
+  bottom: 0px;
```

Comment:
The new actionbar covers the status bar that is being used for development.
<img width="292" alt="image" src="https://user-images.githubusercontent.com/584378/177031399-2e038386-33e0-4cdd-bfd7-68b984bc2b88.png">

Reply:
Maybe I could add some bottom padding? It will ahve to be removed in production.

Reply:
Yes, it will be removed in production, but dev tools are also important to be considered with. Please either render them related to the status bar.